### PR TITLE
Adjustments for core notification and icon changes

### PR DESCRIPTION
**Pull request by bx80**
### Description:

This PR updates the UI test screenshots affected by the notification and icon updates in core PR [#21077](https://github.com/matomo-org/matomo/pull/21077)

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: tests/UI/expected-ui-screenshots/LogViewer_filter_date.png
Comment (michalkleiner) [inline tests/UI/expected-ui-screenshots/LogViewer_filter_date.png:1]: Missing styling completely.

File: tests/UI/expected-ui-screenshots/LogViewer_filter_severity.png
Comment (michalkleiner) [inline tests/UI/expected-ui-screenshots/LogViewer_filter_severity.png:1]: Missing styling completely.

File: tests/UI/expected-ui-screenshots/LogViewer_info_no_supported_writer.png
Comment (michalkleiner) [inline tests/UI/expected-ui-screenshots/LogViewer_info_no_supported_writer.png:1]: It seems like the action icons next to the "Use regular expressions" checkbox needs a bit of a tweak around their alignment, is there perhaps a custom CSS applied?

**Review by michalkleiner — CHANGES_REQUESTED** [review]

File: tests/UI/expected-ui-screenshots/LogViewer_filter_date.png
Comment (bx80) [reply tests/UI/expected-ui-screenshots/LogViewer_filter_date.png:1]: This is a randomly failing tests which has been like this for quite a while, it's nothing to do with the changes in this PR.

File: tests/UI/expected-ui-screenshots/LogViewer_filter_severity.png
Comment (bx80) [reply tests/UI/expected-ui-screenshots/LogViewer_filter_severity.png:1]: Also a randomly failing test unrelated to these changes.

File: tests/UI/expected-ui-screenshots/LogViewer_filter_date.png
Comment (michalkleiner) [reply tests/UI/expected-ui-screenshots/LogViewer_filter_date.png:1]: Yep, that's fine, it was more about making sure we don't commit new UI screenshots that are from a failed run.

**Pull request merged**